packages/wasm/crate/src/storage.rs

```diff
@@ -379,4 +358,39 @@ impl IndexedDBStorage {
 
         Ok(records)
     }
+
+    pub async fn get_auction_oustanding_reserves(
+        &self,
+        auction_id: AuctionId,
+    ) -> WasmResult<OutstandingReserves> {
+        self.db
+            .transaction_on_one(&self.constants.tables.auctions)?
+            .object_store(&self.constants.tables.auctions)?
+            .get::<JsValue>(&byte_array_to_base64(&auction_id.to_proto().inner).into())?
+            .await?
+            .map(|auction| {
+                serde_wasm_bindgen::from_value::<AuctionRecord>(auction)?
+                    .outstanding_reserves
+                    .ok_or(WasmError::Anyhow(Error::msg("could not find reserves")))
```

Comment:
Is there a clean way to not have to repeat this error line below on line 376?

Reply:
these are technically different errors - i would put 'could not find auction' here, and 'could not find reserves' is appropriate at the other.

consider also rewriting it in the pattern of other fns in this file, which seem a bit clearer.